src/constants/index.ts

```diff
@@ -1,9 +1,18 @@
-import { ProducerPartitionerType } from '../interfaces';
+import { logLevel } from 'kafkajs';
+import { Strategy } from '../interfaces';
 
-export const DEFAULT = {
-  autoCommit: false,
-  fetchSizeInMB: 3,
+interface Default {
+  maxMessagesPerTopic: number;
+  onlyTesting: boolean;
+  connectionTTL: number;
+  strategy: Strategy;
+  kafkaJSLogs: logLevel;
+}
+
+export const DEFAULT: Default = {
```

Comment:
what do you think ? 
```suggestion
export const DEFAULT_CONFIG: Default = {
```
because it can be a bit confusing to have a `default` interface and a `default` object, even if it's written in upper case ajajajajaaj